### ISSUE TITLE
'Location Search failed' all providers except AccuWeather

### ISSUE THREAD

**Issue by Rexadev** [breezy-weather/breezy-weather#281]
### Steps to reproduce

Try to add a location
change provider
type some place name like new york

### Expected behavior

Give me location names

### Actual behavior

'Location Search failed' toast

### Weather source used

All expect AccuWeather

### Breezy Weather version

4.4.1-beta

### Android version

Android 11

### Device

crDroid Official vince

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I read the [Frequently Asked Questions](https://github.com/breezy-weather/breezy-weather/blob/main/HELP.md) document and followed instructions if relevant.
- [X] I have updated the app to **[latest version](https://github.com/breezy-weather/breezy-weather/releases/latest)**.

**Comment by papjul** [comment]
AccuWeather and China have their own search endpoint, while all others depend on Open-Meteo.

Can you make a recording of you typing New York and having no result? Thanks!

**Comment by zmx01403137** [comment]
I can reproduce it from China Mobile's network, https://geocoding-api.open-meteo.com/v1/search?name=New%20York&count=10&language=en&format=json is unreachable, the error is like below:
_The connection has timed out
An error occurred during a connection to geocoding-api.open-meteo.com.
    The site could be temporarily unavailable or too busy. Try again in a few moments.
    If you are unable to load any pages, check your computer’s network connection.
    If your computer or network is protected by a firewall or proxy, make sure that Firefox is permitted to access the web._

$ ping geocoding-api.open-meteo.com
_Pinging geocoding-api.open-meteo.com [202.61.206.6] with 32 bytes of data:
Request timed out.
Request timed out.
Request timed out.
Request timed out.
Ping statistics for 202.61.206.6:
    Packets: Sent = 4, Received = 0, Lost = 4 (100% loss),_

$ tracert geocoding-api.open-meteo.com
_Tracing route to geocoding-api.open-meteo.com [202.61.206.6]
over a maximum of 30 hops:
  1     4 ms     3 ms     3 ms  192.168.2.1
  2     4 ms     3 ms     2 ms  192.168.1.1
  3   195 ms    16 ms     4 ms  lo0-100.BSTNMA-VFTTP-361.verizon-gni.net [100.0.0.1]
  4     5 ms     4 ms     4 ms  218.207.152.37
  5     7 ms     8 ms     8 ms  112.5.255.253
  6     7 ms    16 ms     7 ms  111.24.11.33
  7    24 ms    21 ms    22 ms  221.183.96.202
  8    26 ms     *        *     221.183.89.1
  9     *       30 ms    34 ms  221.183.89.34
 10    35 ms    35 ms    35 ms  221.183.89.169
 11   245 ms   246 ms   245 ms  223.120.16.141
 12   244 ms   244 ms   243 ms  223.120.10.13
 13   261 ms   257 ms   257 ms  de-cix1.rt.act.fkt.de.retn.net [80.81.192.73]
 14   326 ms   310 ms   299 ms  GW-Anexia.retn.net [87.245.240.71]
 15   360 ms   320 ms   326 ms  netcup-gw.bbr01.anx84.nue.de.anexia-it.net [144.208.211.31]
 16   258 ms   257 ms   256 ms  dcr02.netcup.net [46.38.252.33]
 17     *        *      330 ms  geocoding-api.open-meteo.com [202.61.206.6]
Trace complete._

**Comment by papjul** [comment]
I don't think Rexadev is from China but if:
- Website is blocked by DNS/VPN/hosts/government/etc
- or Open-Meteo is temporarily unavailable

There is not much we can do unfortunately.

Either way, it's not a bug, and you can retry later or from another network. If no one was able to.find "New York", we would have tons of issues reported.

**Comment by Rexadev** [comment]
Not in china, dns is not blocked I use NextDNS and can verify, tried with/without VPN

@papjul

**Comment by Rexadev** [comment]
> Open-Meteo

Why does OpenWeatherMap and MET Norway depend on Open-Meteo ?
Location service in crDroid and LineageOS uses these 2 and work for me

**Comment by papjul** [comment]
> Not in china, dns is not blocked I use NextDNS and can verify, tried with/without VPN

Can you follow these instructions?

>     1. Can you make a recording of you typing New York and having no result?
> 
>     2. Can you try accessing https://geocoding-api.open-meteo.com/v1/search?name=New%20York&count=10&language=en&format=json from your device?

> > AccuWeather and China have their own search endpoint, while all others depend on Open-Meteo.
> 
> Why does OpenWeatherMap and MET Norway depend on Open-Meteo ? Location service in crDroid and LineageOS uses these 2 and work for me

MET Norway doesn’t have a "location search" feature, so it’s not possible crDroid and LineageOS have a location search feature with this one, and OpenWeather one is garbage and not usable.

**Comment by Rexadev** [comment]
https://github.com/breezy-weather/breezy-weather/issues/329

**Comment by papjul** [comment]
This issue has been left in “Needs more info” mode for the last 7 days and is being automatically marked as “Stale”. If you would like to continue with this issue, please provide any missing information or answer any open questions. If you could resolve the issue yourself meanwhile, please leave a note for future readers with the same problem and close the issue.

If there is no further activity on this issue, it will be closed in a week.

**Comment by papjul** [comment]
Oh, so you didn't type "New York" actually, but you typed a hierarchy, right?

Can we close this issue and do the follow-up in #330?

**Comment by Rexadev** [comment]
@papjul Reopen #329. Close this issue
